test/tests/functional/pbs_resc_used_single_node.py

```diff
@@ -190,44 +191,53 @@ def test_epilogue_single_node(self):
 
         acctlog_match = 'resources_used.foo_f=0.09'
         s = self.server.accounting_match(
-            "E;%s;.*%s.*" % (jid, acctlog_match), regexp=True, n=100)
+            "E;%s;.*%s.*" % (jid, acctlog_match), regexp=True, n="ALL",
```

Comment:
I think you don't need `n="ALL"` when you pass `starttime`, with `starttime` it should work (no matter what internally it does tail/head/cat). With only `starttime` if it is not working then its fw bug

Also, you don't need to pass `starttime` as it's by default to test start time, in other words, to fix this test, you simply need to remove `n=100`, and after that if still failing then its fw bug